src/pipeline/container.go

```diff
@@ -89,3 +89,23 @@ func waitContainer(id string) (int, error) {
 
 	return client.WaitContainer(id)
 }
+
+func isImageLocal(image string) bool {
```

Comment:
This should return (bool, error), so you can return errors from the docker.Client